scipy/optimize/__init__.py

```diff
@@ -253,6 +254,8 @@
 from ._hungarian import linear_sum_assignment
 from ._differentialevolution import differential_evolution
 from ._lsq import least_squares, lsq_linear
+from .shgo_m import *
```

Comment:
Only `shgo` from `._shgo` should be being imported into `__init__.py`

Reply:
Removed with 254c931.